tests/test_settings.py

```diff
@@ -25,3 +25,16 @@ def test_ses_configuration_set_given(self):
         unload_django_ses()
         import django_ses
         self.assertEqual(django_ses.settings.AWS_SES_CONFIGURATION_SET, settings.AWS_SES_CONFIGURATION_SET)
+
+    def test_ses_region_to_endpoint_default_given(self):
+        unload_django_ses()
+        import django_ses
+        self.assertEqual(django_ses.settings.AWS_SES_REGION_NAME, 'us-east-1')
+        self.assertEqual(django_ses.settings.AWS_SES_REGION_ENDPOINT, f'email.{django_ses.settings.AWS_SES_REGION_NAME}.amazonaws.com')
+
+    def test_ses_region_to_endpoint_set_given(self):
+        settings.AWS_SES_REGION_NAME = 'eu-west-1'
+        unload_django_ses()
+        import django_ses
+        self.assertEqual(django_ses.settings.AWS_SES_REGION_NAME, settings.AWS_SES_REGION_NAME)
```

Comment:
Seems a bit safer to check this is `eu-west-1` on the right side of the equality, but I'm OK with it as-is, too. What do you think?

Reply:
@pcraciunoiu I was just following the code tests pattern that was present. But it does make sense to check `REGION_NAME` as `main.eu-west-1.amazonaws.com` rather than placeholder I guess. Thanks for the suggestion.